### PR TITLE
Add  Mockplus to Design and UI

### PR DESCRIPTION
**Pull request by 550406066**
Add  Mockplus to Design and UI
thanks

### CODE REVIEW

**Comment by ripienaar** [comment]
This doesn’t appear to be SaaS? You download and install locally?

**Comment by 550406066** [comment]
> This doesn’t appear to be SaaS? You download and install locally?

Hi Pienaar,

Mockplus is like Adobe XD. It is an all-in-one product platform for prototyping, collaboration, and design systems. We have more than 1,00,0,000 users around the world. 

Mockplus RP is a faster and easier prototyping and design tools. Users can download Mockplus RP locally for design.

Mockplus iDoc is a collaboration & handoff tool for designers to collaborate and handoff a design to other designers, developers and product managers effortlessly. It helps connect the entire product design workflow. It facilitates handoff by taking designs from PS, Sketch, Adobe XD and exporting into a format that can generate code snippets, specs, and assets. designers and developers can use it on the web. 

Yes, it is a SaaS tool and you can try it here: https://www.mockplus.com/mockplus-idoc

**Comment by ripienaar** [comment]
Ok then briefly state what’s included in the free tier like for example in the adobe example one line up

**Comment by 550406066** [comment]
> Ok then briefly state what’s included in the free tier like for example in the adobe example one line up

Free plan features: create interactive prototypes & UI flows; compare, comment and review designs; manage design tasks and resources; inspect and export design specs; access and download assets; etc

**Comment by ripienaar** [comment]
Update the PR please adjusting the comment there with whats the limit. Examples:

```
Free plan covers: 1 active shared design spec, Adobe Fonts Free (limited set of fonts), 2GB of cloud storage.
```

```
No registration required, 15000 queries per hour rate limit.
```

```
Forever free plan for developers with 30k requests per month (1k/day) limit.
```

short, concise and detailing just the main limitations, we cannot take an entire essay so just summarise please and update the PR with the wording

File: README.md [hunk 1]
```diff
@@ -744,6 +744,7 @@ Table of Contents
 ## Design and UI
 
   * [Adobe XD](https://www.adobe.com/products/xd.html) - Wireframe & Prototyping tool similar to Sketch. Free plan covers: 1 active shared design spec, Adobe Fonts Free (limited set of fonts), 2GB of cloud storage.
+  * [Mockplus](https://mockplus.com/) — Mockplus is an all-in-one product platform for prototyping, collaboration, and design systems. It offers the best prototyping tool that can help designers create interactive prototypes in minutes..
```
Comment (550406066) [inline README.md]: * [Mockplus](https://www.mockplus.com/idoc) — Mockplus iDoc is a powerful design collaboration & handoff tool. Free Plan includes 3 users and 5 projects with all features available.

**Comment by 550406066** [comment]
> Update the PR please adjusting the comment there with whats the limit. Examples:
> 
> ```
> Free plan covers: 1 active shared design spec, Adobe Fonts Free (limited set of fonts), 2GB of cloud storage.
> ```
> 
> ```
> No registration required, 15000 queries per hour rate limit.
> ```
> 
> ```
> Forever free plan for developers with 30k requests per month (1k/day) limit.
> ```
> 
> short, concise and detailing just the main limitations, we cannot take an entire essay so just summarise please and update the PR with the wording

Hi Pienaar,

We have changed our content. The Free Plan includes 3 users and 5 projects with all features available.

**Comment by aslafy-z** [comment]
Related: https://github.com/ripienaar/free-for-dev/pull/1429

**Comment by ripienaar** [comment]
thanka

**Pull request closed without merging**